web/js/tasks.js

```diff
@@ -884,6 +884,37 @@ Ext.onReady(function(){
             window.scrollTo(0, taskPanel.getEl().getY());
     }
 
+    window.setInterval(function () {
+        if( unsavedChanges == true ) {
+            var panels = tasksScrollArea.items;
+            var valids = true;
+            for(var panel=0; panel<panels.getCount(); panel++) {
+                console.log();
+                if (!panels.get(panel).initTimeField.isValid() ||
+                    !panels.get(panel).endTimeField.isValid() ||
+                    !panels.get(panel).customerComboBox.getRawValue() ||
+                    !panels.get(panel).projectComboBox.getRawValue() ||
+                    !panels.get(panel).storyField.getRawValue() ||
+                    !panels.get(panel).descriptionTextArea.getRawValue()
+                ) {
```

Comment:
The only compulsory fields are init and end time. Tasks may be saved with empty customer, project, etc.